api/v1beta1/tenant_func.go

```diff
@@ -36,3 +36,12 @@ func (t *Tenant) AssignNamespaces(namespaces []corev1.Namespace) {
 	t.Status.Namespaces = l
 	t.Status.Size = uint(len(l))
 }
+
+func (t *Tenant) GetOwnerProxySettings(name string, kind OwnerKind) []ProxySettings {
```

Comment:
I don't want to be pedantic here but could you we use here a sorted slice in order to take advantage of the binary search?

Reply:
done :)